recipe/windows-bat.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/graphviz/backend.py b/graphviz/backend.py
+index 955551a..be35745 100644
+--- a/graphviz/backend.py
++++ b/graphviz/backend.py
+@@ -69,6 +69,8 @@ def command(engine, format, filepath=None):
+     if format not in FORMATS:
+         raise ValueError('unknown format: %r' % format)
+ 
++    if PLATFORM == 'windows':
++        engine = '%s.bat' % engine
```

Comment:
> The advantage of my approach is that `cmd.exe` blank windows won't be shown
> when making calls to `python-graphviz` from GUIs like Spyder or the Jupyter notebook.

Can you provide a more detailed description where this occured [upstream](https://github.com/xflr6/graphviz/issues)?

AFAIK, command-line windows are suppressed by [these flags](https://github.com/xflr6/graphviz/blob/7f608c1e6a6567dac894e2fe80a921d674273902/graphviz/backend.py#L57-L62), which are used on Windows.